roles/contiv_network/tasks/main.yml

```diff
@@ -20,21 +20,35 @@
 - name: download netmaster and netplugin
   get_url:
     validate_certs: "{{ validate_certs }}"
-    url: https://github.com/contiv/netplugin/releases/download/v0.0.0-12-11-2015.20-54-40.UTC/netplugin-v0.0.0-12-11-2015.20-54-40.UTC.tar.bz2
+    url: https://github.com/contiv/netplugin/releases/download/v0.1-01-28-2016.03-55-05.UTC/netplugin-v0.1-01-28-2016.03-55-05.UTC.tar.bz2
```

Comment:
This release does not seem to have -docker-plugin argument to netplugin. So, this will break when netplugin is deployed. You might have to change the NETPLUGIN_ARGS in roles/contiv_network/templates/netplugin.j2